docs/docs/server-config/backing-up-the-server.md

```diff
@@ -1,13 +1,12 @@
 ---
-layout: article
 title: Backing Up Libretime
-category: admin
 ---
 
-> At the moment, there is not a way to automatically restore a Libretime backup.
-> To restore a failed Libretime instance, install a fresh copy, go through the
-> standard setup process, and reupload the backed-up media files. A _Watched Folders_
-> feature is [currently in development](https://github.com/LibreTime/libretime/issues/70).
+:::info
+
+At the moment, there is no script to cleanly restore a Libretime backup. To restore a Libretime backup, install a fresh copy, go through the standard setup process, and reupload the backed-up media files. A Watched Folders feature is [currently in development](https://github.com/LibreTime/libretime/issues/70).
+
```

Comment:
Not to myself: I don't like this statement, watched folder is not the same as a backup/restore mechanism, we should correct this.